src/gui/dialogs/window_dlg_wait.c

```diff
@@ -101,7 +101,10 @@ void window_dlg_wait_draw(window_dlg_wait_t *window) {
             rect_ui16_t rc_pro = rc;
             rc_pro.x = 10;
             rc_pro.w -= 20;
-            char text[16];
+            // why 16 bytes for "0%" - "100%"? Thats just 5 bytes instead of 16
+            // Save stack, guys!
+            static const uint32_t text_size = 5;//16;
```

Comment:
/another nitpick/
Again, let's move it to a commit message :)

Reply:
but then probably no one would see it while reading the code ;)